### ISSUE TITLE
libbb10styleplugin.so "file does not exist" error

### ISSUE THREAD

**Issue by panmarco83** [Maldela/fancontrol-gui#41]
I thought this was something hard coded that was causing this error but there is no reference to 'libbb10styleplugin' in your code anywhere..
```
CMake Error at /usr/lib/x86_64-linux-gnu/cmake/Qt5Widgets/Qt5WidgetsConfig.cmake:27 (message):
  The imported target "Qt5::Widgets" references the file

     "/usr/lib/x86_64-linux-gnu/qt5/plugins/styles/libbb10styleplugin.so"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.
```
I've been trying to figure out where it comes from but I can't find said module on my system or in apt.
these are the ones I do have in the same folder that cmake is looking into:
```
adwaita.so
libkvantum.so
libqcleanlooksstyle.so
libqgtk2style.so
libqmotifstyle.so
libqplastiquestyle.so
libqt5ct-style.so
```
Ubuntu 20.04 on Cinnamon desktop

**Comment by Maldela** [comment]
> I thought this was something hard coded that was causing this error but there is no reference to 'libbb10styleplugin' in your code anywhere..
Seems like your Qt5::Widgets package may be broken. Have you tried repairing/reinstalling Qt?

**Comment by panmarco83** [comment]
Yes, to no avail.
Another thing is that ```libbb10styleplugin.so``` is a Black Berry 10 theme.
Qt needs to be compiled with BB10 support for this to be included.

Are you on Arch or Gentoo by any chance ?

**Comment by Maldela** [comment]
I'm on Arch. Is Qt5::Widgets different on Arch and Gentoo than on Ubuntu?

**Comment by panmarco83** [comment]
Don't know, but they are quite possibly compiled with different options I suppose.

**Comment by Maldela** [comment]
I don't really know how to help you here. Maybe ask on the Ubuntu forums?
My last attempt at fixing this would be editing `/usr/lib/x86_64-linux-gnu/cmake/Qt5Widgets/Qt5WidgetsConfig.cmake` and deleting the part that references `libbb10styleplugin.so`.

**Comment by panmarco83** [comment]
There is no mention of it:  
```
source@megumi:~$ cat /usr/lib/x86_64-linux-gnu/cmake/Qt5Widgets/Qt5WidgetsConfig.cmake|grep -i libbb10styleplugin
source@megumi:~$ 
```

**Comment by Maldela** [comment]
But your original error message says `/usr/lib/x86_64-linux-gnu/cmake/Qt5Widgets/Qt5WidgetsConfig.cmake` references `/usr/lib/x86_64-linux-gnu/qt5/plugins/styles/libbb10styleplugin.so` on line 27.

**Comment by panmarco83** [comment]
Interesting, I moved `/usr/lib/x86_64-linux-gnu/cmake/Qt5Widgets/Qt5Widgets_QBB10StylePlugin.cmake` away and tried to run cmake again, now it works..  
My apologies, I had fixated on this being something on your end.  
What I can tell is that the file might be an errant file from some package I have not figured out the origins of.  
At the very least you might be able to use this issue as a possible solution if others bump into the same problem.

- Regards
